docs/plugin-protocol/tfplugin5.proto

```diff
@@ -811,3 +819,45 @@ message CallFunction {
         FunctionError error = 2;
     }
 }
+
+message ListResource {
+    message Request {
+        // type_name is the managed resource type name.
+        string type_name = 1;
+
+        // configuration is the list ConfigSchema-based configuration data.
+        DynamicValue config = 2;
+    }
+
+    message Event {
+        // Resource contains all data associated with each resource instance.
+        message Resource {
+            // identity is the resource identity data of the resource instance.
+            ResourceIdentityData identity = 1;
```

Comment:
nit: IdentityData instead of ResourceIdentityData. The nested message is already called `Resource`

Reply:
Since the type `ResourceIdentityData` is already used in other parts of the protocol, I think we shouldn't rename it. But if we rename the message to `Result` the duplication is solved as well